linux_files/setup

```diff
@@ -92,40 +86,56 @@ function byemessage {
 ## internationalization
 
 function languageprompt {
-read -r -p "Would you like to configure additional languages? [y/N] " response
-if [[ "$response" =~ ^[yY]$ ]]
-then
+if (whiptail --title "Language" --yesno "Would you like to configure for international languages?" 8 45) then
     echo "Running $ dpkg-reconfigure locales"
     sudo dpkg-reconfigure locales
 fi
 }
 
-## package meta functions
-
-    # list all package sets and the order they should be presented for optional install here
-
-function installpackages {
-echo "Shells:"
-installandsetshell
-echo "Editors:"
-neoviminstall
-emacsinstall
-echo "Enhancements:"
-fzfinstall
-explorerintegration
-## package set functions
-echo "Development Environment:"
-pythoninstall
-nodeinstall
-goinstall
-powershellinstall
-echo "Cloud Tools:"
-azurecliinstall
-echo "GUI Applications:"
-basicguiinstall
-codeinstall
-chromeinstall
-hidpiconfig
+## package index
+
+function installmenu {
+
+choice=$(whiptail --title "WLinux Setup" --checklist --separate-output "Choose:" 20 100 12 \
+"EXPLORER" "Enable right-click on folders in Windows Explorer to open them in WLinux" off \
+"SHELLS" "Install and configure default shells from bash, zsh, csh, and fish" off \
+"EDITORS" "Install text editors neovim, emacs, or Visual Studio Code (requires X)" off \
+"PYTHONPI" "Install Python 3.7 and download and install latest PyPi" off \
+"NODEJS" "Install Node.js and npm" off \
+"GO" "Install the latest Go" off \
+"POWERSHELL" "Install PowerShell for Linux and Azure CLI tools" off \
+"GUI" "Install the basic libraries needed for most GUI apps" off \
+"LANGUAGE" "Change language default languane and keyboard set in WSL" off \
+3>&1 1>&2 2>&3)
+
+while read choice
+do
+        case $choice in
+                Explorer) explorerintegration
```

Comment:
I believe the case switch in bash is case sensitive... I cannot run the part here